app/views/incoming_trusts/show.html.erb

```diff
@@ -69,3 +69,7 @@
     </div>
   </dl>
 </div>
+
+<div class="govuk-width-container">
+  <%= link_to t(".next_action_link"), trust_projects_path(params[:trust_id]), method: :post, class: "govuk-button" %>
```

Comment:
Would it make sense to set an `@trust_id` instance variable in the `#show` action rather than referencing via `params[:trust_id]` ?

Reply:
Thank you - that's made me rethink how this was done.

I've added an instance variable `@outgoing_trust_id` as that makes it clearer how it related to the other trust.